AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2551,6 +2551,11 @@ - (void)_insertSubnode:(ASDisplayNode *)subnode atSubnodeIndex:(NSInteger)subnod
     return;
   }
   
+  if (self.layerBacked && !subnode.layerBacked) {
+    ASDisplayNodeFailAssert(@"Cannot add a view-backed node as a subnode of a layer-backed node. Supernode: %@, subnode: %@", self, subnode);
```

Comment:
yeah, this is really weird.  The check did exist before, but I'm not sure how long ago.  We should be pretty worried it vanished lol.